src/elm/Model.elm

```diff
@@ -200,8 +200,7 @@ updateReadyModel msg model =
                     RemoteUser.UserLoaded user ->
                         case user of
                             Anonymous _ ->
-                                -- TODO: trigger modal popup here
-                                Utils.simpleUpdate model
+                                ( model, Utils.toCmd <| AuthenticationButtonClicked AuthenticationInfo.SignUp )
```

Comment:
I am a strong advocate of having events / `msg`s in Elm apps reflect what a user has done. 

If we emit a `AuthenticationButtonClicked` event when in reality they clicked the vote button, things can get complicated and hard to follow quickly.

My suggestion is to abstract away the logic that handles the `AuthenticationButtonClicked` event.

https://github.com/parlez-vous/embed/blob/master/src/elm/Model.elm#L284

Reply:
Right. Will fix this.

Reply:
@supermacro, I've fixed this changes. Can you review it again?